package.json

```diff
@@ -14,26 +14,32 @@
   "dependencies": {
     "@jsdevtools/rehype-toc": "^3.0.2",
     "@svgr/webpack": "^6.5.0",
+    "@typescript-eslint/parser": "^7.7.1",
     "classnames": "^2.3.2",
     "framer-motion": "^7.6.7",
     "gray-matter": "^4.0.3",
     "image-size": "^1.0.2",
     "jwt-decode": "^3.1.2",
     "lodash.clonedeep": "^4.5.0",
     "minimatch": "^5.1.2",
-    "next": "13.0.0",
-    "next-image-export-optimizer": "^1.0.1",
+    "next": "^14.2.3",
```

Comment:
We upgraded the next version in this PR, is there a reason? In my opinion, it would be better to do it in a separate PR if it is not necessary.

Reply:
Because the Yorkie UI library used the version next > 14 then I updated the version.